client/src/components/common/names-capture.vue

```diff
@@ -3,7 +3,7 @@
     <v-container fluid class="pa-0" id="send-to-examination-container">
       <template v-if="editMode">
         <v-row justify="end" class="mt-n3 mb-n5">
-          <v-col cols="6" class="copy-normal">I need a name to:</v-col>
+          <v-col cols="6" class="font-weight-bold">I need a name to:</v-col>
```

Comment:
Now you've lost the "copy-normal" styles. Is that OK?

Reply:
You bet, it is inherited here from container.